src/user/user.resolver.ts

```diff
@@ -0,0 +1,27 @@
+import { Args, Mutation, Query, Resolver } from '@nestjs/graphql';
+import { User } from './entities/user.entity';
+import { UserService } from './user.service';
+import { CreateUserDto } from './dto/create-user.dto';
+
+@Resolver(() => User)
+export class UserResolver {
+  constructor(private readonly userService: UserService) {}
+
+  @Query(() => [User],)
+  async users(): Promise<User[]> {
+    return this.userService.findAll();
+  }
+  @Mutation(() => User)
+  createUser(
+    @Args('firstName') firstName: string,
+    @Args('lastName') lastName: string,
+    @Args('age') age: number,
+  ){
+    const createUserDto = new CreateUserDto();
```

Comment:
@Nandita-Nikam  you should use  `@InputType` when there are more than one params in the input

Reply:
ok